xml/System.Security.Claims/ClaimsIdentity.xml

```diff
@@ -1555,9 +1586,11 @@
         <Parameter Name="writer" Type="System.IO.BinaryWriter" />
       </Parameters>
       <Docs>
-        <param name="writer">To be added.</param>
-        <summary>To be added.</summary>
-        <remarks>To be added.</remarks>
+        <param name="writer">The <see cref="T:System.IO.BinaryWriter" /> to use for data storage.</param>
```

Comment:
since the type of the param is displayed on the page, you can use a generic language like The writer...